kmsconfig/config_test.go

```diff
@@ -117,7 +117,9 @@ func TestConfig(t *testing.T) {
 					TestString      string        `config:"test_string"`
 					TestStringSlice []string      `config:"test_string_slice"`
 					TestInt         int64         `config:"test_int"`
-					TestTime        time.Duration `config:"test_time"`
+					TestTimeSecond  time.Duration `config:"test_time" config_duration_type:"seconds"`
```

Comment:
Can the struct tags be formatted like this? https://github.com/vidsy/go-models/blob/master/api/data/brief.go#L14